modules/gitops-customizing-permissions-for-cluster-scoped-instances.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assembly:
+//
+// * declarative_clusterconfig/customizing-permissions-by-creating-user-defined-cluster-roles-for-cluster-scoped-instances.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-customizing-permissions-for-cluster-scoped-instances.adoc{context}"]
+= Customizing permissions for cluster-scoped instances
+
+As a cluster administrator, to customize permissions for cluster-scoped instances, you must create new cluster roles and cluster role bindings for the {gitops-shortname} control plane components. 
+
+For example purposes, the following instructions focuses only on user-defined cluster-scoped instances.
+
+.Procedure
+
+. Open the *Administrator* perspective of the web console and go to *User Management* -> *Roles* -> *Create Role*. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
```

Comment:
This is a lot of actions for a single step. Consider making the second sentence ("Use the following `ClusterRole`...") a separate step.